packages/icons/src/PentahoIconWrapper.tsx

```diff
@@ -0,0 +1,90 @@
+import styled from "@emotion/styled";
+import { SizeProp } from "@fortawesome/fontawesome-svg-core";
+import {
+  FontAwesomeIcon,
+  FontAwesomeIconProps,
+} from "@fortawesome/react-fontawesome";
+import { getColor, HvColorAny, HvSize } from "@hitachivantara/uikit-styles";
+
+import { IconSize, svgSizeMap } from "./IconBase";
+
+export interface PentahoIconWrapperProps
+  extends Omit<FontAwesomeIconProps, "color" | "size"> {
+  size?: HvSize;
+  iconSize?: IconSize;
+  color?: HvColorAny;
+  container?: boolean;
+}
+
+const getContainerSize = (iconSize?: IconSize | HvSize) => {
+  if (!iconSize) return;
+  const baseSize = svgSizeMap[iconSize] || svgSizeMap.sm;
+  // container has 8px margin on each side, except for XS which has 10px
+  const containerSize =
+    baseSize + 2 * (iconSize === "XS" || iconSize === "xs" ? 10 : 8);
+
+  return { width: containerSize, height: containerSize };
+};
+
+const getSVGSize = (iconSize?: IconSize | HvSize) => {
+  if (!iconSize) return;
+  const baseSize = svgSizeMap[iconSize] || svgSizeMap.sm;
+  return { width: baseSize, height: baseSize };
+};
+
+const StyledIconContainer = styled("div")({
+  display: "flex",
+  justifyContent: "center",
+  alignItems: "center",
+});
+
+export const PentahoIconWrapper = ({
```

Comment:
I really don't like the name `Wrapper` 🤷 
Sometimes it's just a container (and `Container` is a clearer name). But most of the times it's more than a container, and the name is redundant & confusing

Here's it's just unnecessary. And we're in need of some docs

```suggestion
/** DOCS! */
export const PentahoIcon = ({
```